back/src/domains/convention/use-cases/ConventionsReminder.ts

```diff
@@ -62,52 +62,33 @@ export class ConventionsReminder extends TransactionalUseCase<
     uow: UnitOfWork,
   ): Promise<ConventionsReminderSummary> {
     const now = this.#timeGateway.now();
-    const [
-      conventionsForLastSignatoryReminder,
-      conventionsForAgencyReminders,
-      conventionsForFirstSignatoryReminder,
-    ] = await Promise.all([
-      uow.conventionQueries.getConventions({
-        filters: {
-          startDateGreater: now,
-          startDateLessOrEqual: addBusinessDays(now, TWO_DAYS),
-          withStatuses: signatoryStatuses,
-        },
-        sortBy: "dateStart",
-      }),
-      uow.conventionQueries.getConventions({
-        filters: {
-          startDateGreater: now,
-          startDateLessOrEqual: addBusinessDays(now, THREE_DAYS),
-          withStatuses: agencyStatuses,
-        },
-        sortBy: "dateStart",
-      }),
-      uow.conventionQueries.getConventions({
-        filters: {
-          dateSubmissionEqual: subDays(now, TWO_DAYS),
-          withStatuses: [...signatoryStatuses],
-        },
-        sortBy: "dateStart",
-      }),
-    ]);
+    const [conventionsForLastSignatoryReminder, conventionsForAgencyReminders] =
+      await Promise.all([
+        uow.conventionQueries.getConventions({
+          filters: {
+            startDateGreater: now,
+            startDateLessOrEqual: addBusinessDays(now, TWO_DAYS),
+            withStatuses: signatoryStatuses,
+          },
+          sortBy: "dateStart",
+        }),
+        uow.conventionQueries.getConventions({
+          filters: {
+            startDateGreater: now,
+            startDateLessOrEqual: addBusinessDays(now, THREE_DAYS),
+            withStatuses: agencyStatuses,
+          },
+          sortBy: "dateStart",
+        }),
+      ]);
```

Comment:
Vu que ce usecase peut avoir des soucis de surcharge en DB, ça vaudrait le coup de charger les conventions en 2 fois au lieu du Promise.all().

```suggestion
    const conventionsForLastSignatoryReminder = await uow.conventionQueries.getConventions({
          filters: {
            startDateGreater: now,
            startDateLessOrEqual: addBusinessDays(now, TWO_DAYS),
            withStatuses: signatoryStatuses,
          },
          sortBy: "dateStart",
        });
        
    const conventionsForAgencyReminders = await uow.conventionQueries.getConventions({
          filters: {
            startDateGreater: now,
            startDateLessOrEqual: addBusinessDays(now, THREE_DAYS),
            withStatuses: agencyStatuses,
          },
          sortBy: "dateStart",
        });
```